### ISSUE TITLE
Files should show how big they are

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#2429]
### Problem

Currently when viewing a file attachment it doesn't tell you how big the attachment is:

![Screenshot 2024-11-20 at 11 53 08 AM](https://github.com/user-attachments/assets/4923e2eb-fe54-4cf5-ae43-99c6d03aad44)

### Solution

Display file size info (e.g. "7KB", "10MB", etc.) somewhere here:

![Screenshot 2024-11-20 at 11 53 08 AM](https://github.com/user-attachments/assets/5379ace6-deea-4624-a838-ca42dfca0506)

**Comment by SebinSong** [comment]
NOTE to myself:
Once PR #2428 is merged, the original file sizes of images change if they are compressed. This needs to be taken into account and be worked on as another PR.

**Comment by taoeffect** [comment]
@SebinSong The compression should happen before any file is uploaded, not afterward. Once files are uploaded their size shouldn't change... so I'm confused by your comment.

**Comment by SebinSong** [comment]
@taoeffect I mean, the attachment `size` data should be updated accordingly... not compression itself. (Of course before uploading.)

**Comment by SebinSong** [comment]
closed by #2435
